dluhc-component-library/src/components/policyForm/PolicyForm.tsx

```diff
@@ -16,6 +17,34 @@ import {
 import { FormState, FormValue } from "./types";
 import MultiItem from "./components/MultiItem";
 
+const textInput = (
+  value: string,
+  handleValueChange: (index: number, newValue: string) => void,
+  index: number,
+) => {
+  return (
+    <input
```

Comment:
I know this has just been copied over, but could we use our `Input` component for this? Like how we've used the custom `Textarea` below.